<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddServiceDialog</class>
 <widget class="QWidget" name="AddServiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="DLabel" name="label_4">
       <property name="text">
        <string>请求体</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DLineEdit" name="requestBody"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="DLabel" name="label">
         <property name="text">
          <string>名称</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DLineEdit" name="serviceName"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="DLabel" name="label_2">
       <property name="text">
        <string>方法</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="methodCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>GET</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>POST</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="DLabel" name="label_5">
       <property name="text">
        <string>检查方式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DLabel" name="label_9">
       <property name="text">
        <string>间隔</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DSpinBox" name="checkGap">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>秒</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="DLabel" name="label_6">
       <property name="text">
        <string>超时时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DSpinBox" name="timeoutInSec">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DLabel" name="label_7">
       <property name="text">
        <string>秒</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="DLabel" name="label_3">
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DLineEdit" name="url"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>serviceName</tabstop>
  <tabstop>url</tabstop>
  <tabstop>methodCombo</tabstop>
  <tabstop>requestBody</tabstop>
  <tabstop>checkGap</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>timeoutInSec</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
